dask/dataframe/core.py

```diff
@@ -3134,6 +3165,62 @@ def memory_usage(self, index=True, deep=False):
         )
         return delayed(sum)(result.to_delayed())
 
+    @derived_from(pd.Series)
+    def agg(self, arg):
+        numeric_only_aggs = ["prod", "var", "std", "mean"]
+        other_aggs = ["any", "sum", "min", "max", "count"]  # size
+        supported_funcs = numeric_only_aggs + other_aggs
+
+        if isinstance(arg, str):
+            if arg in supported_funcs:
+                return getattr(self, arg)()
+            else:
+                raise ValueError(
+                    "unknown/unsupported aggregate function: {}".format(arg)
+                )
+        elif isinstance(arg, Iterable):
+            supported_aggs = [argfunc in supported_funcs for argfunc in arg]
+            if all(supported_aggs):
+                from .multi import concat
+
+                series = []
+                for aggfunc in arg:
+                    if aggfunc == "size":
```

Comment:
`"size"` is not in the set of supported funcs above, this branch won't be hit (I saw your comment about an issue with this, just notating it so it's not forgotten in future reviews).